src/main/cc/wfa/virtual_people/core/model/utils/consistent_hash.cc

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "wfa/virtual_people/core/model/utils/consistent_hash.h"
+
+#include <cstdint>
+
+namespace wfa_virtual_people {
+
+// This implementation is a copy and formatting of Figure 1 on page 2 in the
+// published paper:
+//   https://arxiv.org/pdf/1406.2294.pdf
+int32_t JumpConsistentHash(uint64_t key, int32_t num_buckets) {
+  int64_t b = 1, j = 0;
```

Comment:
`b = -1` in the paper, but apparently this doesn't matter

Reply:
Fixed.